TACC-Cloud/hazmapper
# task/WG-6  Add arcgis & wms layer support

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [WG-6](https://tacc-main.atlassian.net/browse/WG-6)

## Summary of Changes: ##

- Added support for wms layer using WMSTileLayer from react-leaflet library
- Added support for arcgis layer using TiledMapLayer from react-esri-leaflet library
- Added data to tileServerLayers fixture with arcgis and wms layers 

## Testing Steps: ##
1. Run hazmapper locally and navigate to `localhost:4200/project`
2. Ensure that the map and the argis layers render
Note: The layer is hard to see due to lack of layer functionality in the react map. A screenshot below shows where to zoom in to be able to see the layers

## UI Photos:

<img width="811" alt="Screen Shot 2024-02-12 at 1 10 00 PM" src="https://github.com/TACC-Cloud/hazmapper/assets/23081932/4bc987e2-9c22-4fcf-8af0-f4d3718076d5">

<img width="811" alt="Screen Shot 2024-02-12 at 1 10 18 PM" src="https://github.com/TACC-Cloud/hazmapper/assets/23081932/35516f9a-3d2b-4108-9502-b425e08da410">

<img width="811" alt="Screen Shot 2024-02-12 at 1 10 58 PM" src="https://github.com/TACC-Cloud/hazmapper/assets/23081932/78d8c7ab-2fa0-40ac-b375-736ce17e44cf">
